omnibus/config/projects/chef-server.rb

```diff
@@ -97,6 +97,7 @@
 dependency "private-chef-upgrades"
 dependency "private-chef-cookbooks"
 dependency "chef-ha-plugin-config"
+dependency "erlang-crypto2"
```

Comment:
Let's move this further up.  My guess is this code will not be changing often after we have it stable.